src/utils/Logging.h

```diff
@@ -14,6 +14,13 @@
 #include <iostream>
 #include <string>
 
+#define ESC_NORMAL "\e[0m"
+#define ESC_DIM "\e[2m"
+#define ESC_UNSET "\e[39m"
+#define ESC_RED "\e[91m"
+#define ESC_BLUE "\e[94m"
+#define ESC_YELLOW "\e[93m"
+
 namespace rdma {
 
```

Comment:
I like colored output, but in logfiles this might be more problematic. Not all readers can parse the color.

Reply:
Yeah, the solution to that would be to integrate `terminfo` to check whether the program/terminal attached stderr supports color, but that's way outside the scope of this original PR.